resources/dicts/patrols/general/medcat.json

```diff
@@ -57,13 +57,14 @@
     "success_text": [
         "They have an in-depth conversation about r_c's vision as they walk. By the time they head back to camp, they have a good idea of what they should do with the information.",
         "p_l takes r_c aside, giving them the full benefits of p_l's experience as they discuss r_c's vision. As a medicine cat, they don't take the responsibility lightly, and both of them work together to divine the vision's meaning.",
-        "p_l chooses to pad pelt to pelt with r_c, the spiritual medicine cat gently guiding the conversation. r_c feels deeply privileged and far more at peace after confiding in them, knowing that c_n is in safe paws."
+        "s_c chooses to pad pelt-to-pelt with r_c, the spiritual medicine cat gently guiding the conversation. r_c feels deeply privileged, and far more at peace, after confiding in them, and they know that c_n is in safe paws."
     ],
     "fail_text": [
         "p_l shakes their head sadly. They can't make sense of the vision.",
-        "r_c notices p_l looking at them enviously, and withdraws from the group, dismissing the dream as unimportant."
+        "s_c notices p_l looking at them enviously, and withdraws from the group, dismissing the dream as unimportant."
```

Comment:
does this need the "rc_has_stat" tag?

EDIT: actually, since the success outcome is p_l as s_c and the fail outcome is r_c as s_c, I'm going to code in a way for "rc_has_stat" to be outcome specific so that these don't need to be rewritten.  I'll follow the same convention as other outcome specific tags with "rc_has_stat#", the # being the relevant outcome index.  

Sorry to ask you this ^^; , but could you go through the other "rc_has_stat" patrols (not just the med ones) and fix their tags to the new system? There shouldn't be all that many of them.